modules/acquia_lift_publisher/config/schema/acquia_lift_publisher.entity_config.yml

```diff
@@ -0,0 +1,20 @@
+# Schema for the configuration files of the Acquia Lift Publisher module.
```

Comment:
Good catch adding this. Surprised this yml file isn't already in acquia_lift_publisher